packages/libs/error-stack/src/fmt/mod.rs

```diff
@@ -126,11 +126,11 @@
 //! [`SpanTrace`]: tracing_error::SpanTrace
 //! [`error_stack::fmt::builtin_debug_hook_fallback`]: crate::fmt::builtin_debug_hook_fallback
 //! [`atomic`]: std::sync::atomic
-// This makes sure that `Emit` isn't regarded as dead-code even though it isn't exported on no-std.
-// This just simplifies maintenance, as otherwise we would be in cfg hell.
+// This makes sure that `Diagnostics` isn't regarded as dead-code even though it isn't exported on
+// no-std. This just simplifies maintenance, as otherwise we would be in cfg hell.
 #![cfg_attr(not(feature = "std"), allow(dead_code))]
```

Comment:
could you please elaborate?

Reply:
I don't know if this comment still applies, because we don't have `Diagnostics` anymore. I was wondering, if we could get around the `dead_code` lint.

Reply:
fixed in [6bb122a](https://github.com/hashintel/hash/pull/976/commits/6bb122a3cf9163049ca83c9aab5cc0f70a09f8e5)

(can't remove because `Defer` isn't used, but moved deadcode)